routes/performance/controllers.js

```diff
@@ -0,0 +1,58 @@
+import PerformanceService from "../../services/performance.js";
+import httpResponse from "../../utils/httpResponse.js";
+
+const controller = {
+  add: async (req, res) => {
+    const addResponse = await PerformanceService.add(req.body);
+    if (addResponse.message === "success") {
+      return httpResponse.CREATED(res, addResponse.data);
```

Comment:
Where is the status code for each request?

Reply:
Sir in httpResponse file all the status codes are mention there, we just call the function related to the specific status code.
e.g for status code 201 CREATED() method we are using.